### DISCUSSION TITLE
rAudio with LMS and squeezelite

### DISCUSSION THREAD

**Discussion by sadays-ymm** [rern/rAudio#1418] [Tips]
Dear rAudio user;

quatmo (nickname) added lms and squuezelite to rAudio on hdvietnam forums

link to LMS

https://www.hdvietnam.me/threads/raspberry-pi-rune-audio-mot-trai-nghiem-nghe-nhac-moi.1174389/post-10877144

link to squeezelite

https://www.hdvietnam.me/threads/raspberry-pi-rune-audio-mot-trai-nghiem-nghe-nhac-moi.1174389/post-10877350

The perfect sound of the rAudio became even more perfect

**Comment by lovehifi** [comment]
LMS and Squeezelite (32bit) for rAudio (Arch Linux)
Pi 2 and Pi 3
https://github.com/lovehifi/raudiolms-32bit

**Comment by Raitsa** [comment]
They are running Tidal natively which would be a big plus for many of us.

https://i.imgur.com/0ehueaG.png
